pandas/core/resample.py

```diff
@@ -984,6 +984,30 @@ def asfreq(self, fill_value=None):
         """
         return self._upsample("asfreq", fill_value=fill_value)
 
+    def mean(
+        self,
+        numeric_only: bool | lib.NoDefault = lib.no_default,
+        *args,
+        **kwargs,
+    ):
+        """
+        Compute mean of groups, excluding missing values.
+
+        Parameters
+        ----------
+        numeric_only : bool, default False
+            Include only `float`, `int` or `boolean` data.
+
+            .. versionadded:: 1.5.0
```

Comment:
Why this versionadded?

Reply:
In `var`, `std` method, numeric_only versionadded is written. So I thought `mean` method also need versionadded. If it is wrong, I will fix it.

Reply:
var and std did not support this prior to 1.5, but this function was consistent with the groupby op that already supported this before. So you can remove it